### PR TITLE
Tracy verb to proc

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Made the Tracy profiler verb into a proc, this should (hopefully) correctly hide it now, as it's the only difference I found from others in its category.

Security/Bugfix.

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!security

**Comment by FluffyGhoster** [comment]
!bugfix

**Comment by FluffyGhoster** [comment]
!review

The runtime check error is the classic ZAS active edges moment.

File: html/changelogs/Fluffyghost-tracyverbtoproc.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - security: "Made the Tracy profiler verb into a proc, this should (hopefully) correctly hide it now, as it's the only difference I found from others in its category."
```
Comment (SleepyGemmy) [inline html/changelogs/Fluffyghost-tracyverbtoproc.yml:41]:
```suggestion
  - security: "Made the Tracy profiler verb into a proc."
```

Comment (FluffyGhoster) [reply html/changelogs/Fluffyghost-tracyverbtoproc.yml]: Soulles, but understandable

**Review by SleepyGemmy — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**